export_dagmc_cmd/MyPlugin.cpp

```diff
@@ -10,7 +16,15 @@ MyPlugin::~MyPlugin()
 std::vector<std::string> MyPlugin::get_keys()
 {
   std::vector<std::string> keys;
+#ifdef BUILD_DAGMC_EXPORT
   keys.push_back("DAGMCExportCommand");
+#endif
+#ifdef BUILD_IGEOM_TESTS
+  keys.push_back("iGeom_test");
+#endif
+#ifdef BUILD_MCNP_IMPORT
+  keys.push_back("MCNPImp");
+#endif
 
```

Comment:
I'm wondering if in the future it is going to be important to have each of these capabilities (or at least the importer/exporter) be entirely separate plugins registered with Trelis for the purpose of creating GUI elements that interact with their respective plugins. Perhaps we can cross that bridge when we come to it though.